probe_scraper/parsers/third_party/shared_telemetry_utils.py

```diff
@@ -15,10 +15,12 @@
 # to record from.
 KNOWN_PROCESS_FLAGS = {
     'all': 'All',
-    'all_childs': 'AllChilds',
+    'all_children': 'AllChildren',
     'main': 'Main',
     'content': 'Content',
     'gpu': 'Gpu',
+    # Historical Values
+    'all_childs': 'AllChildren', # Supporting files from before bug 1363725
```

Comment:
Great, this keeps this backwards compatible.
Can you add test coverage? The files to change are tests/test_histograms_parser.py and tests/Histograms.json.